detox/e2e/test/channel_info/channel_notification_preference.e2e.js

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Same here about moving to `detox/e2e/test/notifications` folder.

Reply:
Updated